.github/workflows/docker-build.yaml

```diff
@@ -73,21 +85,17 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           context: .
-          push: true
+          push: ${{ github.event_name == 'push' }}
           provenance: false
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new,mode=max
```

Comment:
If we are using type=local how is it going to be reused for the next build? type=gha is probably a better option

Reply:
We only want to push the image on a `push` event type; therefore, `type=gha` cannot be used here because it requires an external container registry to push to.